app/src/main/java/com/example/lhi06/residue/Album.java

```diff
@@ -2,11 +2,18 @@
 
 import android.support.annotation.NonNull;
 
+import com.google.gson.annotations.SerializedName;
+
 public class Album {
 
-    private final String collectionName;
-    private final String artistName;
-    private final String artworkUrl100;
+    @SerializedName("collectionName")
+    private String collectionName;
+
+    @SerializedName("artistName")
+    private String artistName;
+
+    @SerializedName("artworkUrl100")
+    private String artworkUrl100;
```

Comment:
I would rename this field to `artworkUrl`. The `100` at the end makes no sense in this context